tests/tabular_output/test_output_formatter.py

```diff
@@ -49,3 +69,25 @@ def hello_world(data, headers, **_):
     assert expected == TabularOutputFormatter().format_output(
         data, headers, format_name='ascii', preprocessors=(hello_world,),
         missing_value='hello')
+
+
+def test_format_name_attribute():
+    """Test the the format_name attribute be set and retrieved."""
+    formatter = TabularOutputFormatter(format_name='plain')
+    assert formatter.format_name == 'plain'
+    formatter.format_name = 'simple'
+    assert formatter.format_name == 'simple'
```

Comment:
Lol, I can see what you mean. The reason I added the test is because the `format_name` attribute setter is a custom function that also checks against the supported formats (and raises a ValueError when needed: https://github.com/dbcli/cli_helpers/blob/master/cli_helpers/tabular_output/output_formatter.py#L72:L84

This way, we ensure that code doesn't break and we get 100% code coverage.